key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -381,8 +382,15 @@ primitive_term:
   | ifThenElseTerm
   | ifExThenElseTerm
   | abbreviation
-  | accessterm
-  | literals
+  | prim_conflicting;
```

Comment:
This was a refactoring to locate the ambiguity between `accessterm` and `literals`. 

The answer is in `INT_LITERAL` in the rule `simple_ident_dots`.

Reply:
Will be reverted before merge.